src/test/java/org/openmrs/contrib/isanteplus/qaframework/automation/FindPatientSteps.java

```diff
@@ -0,0 +1,91 @@
+package org.openmrs.contrib.isanteplus.qaframework.automation;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.isanteplus.qaframework.RunTest;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.FindPatientPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.HomePage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.page.LoginPage;
+import org.openmrs.contrib.isanteplus.qaframework.automation.test.TestBase;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class FindPatientSteps extends TestBase {
+	
+	private FindPatientPage findPatientPage;
+	
+	private ClinicianFacingPatientDashboardPage dashboardPage;
+	
+	private LoginPage loginPage;
+	
+	private HomePage homePage;
+	
+	@Before(RunTest.HOOK.FINDPATIENT)
+	public void systemLogin() {
+		homePage = loginPage.goToHomePage();
+		
+	}
+	
+	@After(RunTest.HOOK.FINDPATIENT)
+	public void destroy() {
+		quit();
+	}
+	
+	@Given("User clicks on Find Patient App")
+	public void visitFindPatientPage() {
+		if (findPatientPage != null) {
+			findPatientPage = (FindPatientPage) homePage.goToFindPatientRecord().waitForPage();
```

Comment:
an also , you methods should reflect he step definition ie 
homePage.clickFindPatientRecordApp()